source/usage-examples/findOne.rst

```diff
@@ -0,0 +1,28 @@
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can find a single document using the ``collection.findOne()``
```

Comment:
Can we link `collection.findOne()` to the api reference docs?
https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#findOne